src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String SPLITTER = "&";
+    private static final String SUPPLY = "supply";
+    private static final int OPERATION_CELL = 0;
+    private static final int AMOUNT_CELL = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        StringBuilder builder = new StringBuilder();
+        String[] tableRow;
+        String[] data = readFile(new File(fromFileName));
+        for (int i = 0; i < data.length; i++) {
+            tableRow = data[i].split(",");
+            if (tableRow[OPERATION_CELL].equals(SUPPLY)) {
+                totalSupply += Integer.parseInt(tableRow[AMOUNT_CELL]);
+            } else {
+                totalBuy += Integer.parseInt(tableRow[AMOUNT_CELL]);
+            }
+        }
+        builder.append("supply").append(",").append(totalSupply).append(System.lineSeparator())
```

Comment:
You even already have one)
```suggestion
        builder.append(SUPPLY).append(",").append(totalSupply).append(System.lineSeparator())
```